.tooling/lua/tests/run.sh

```
@@ -3,14 +3,14 @@
 # output is saved to file and error will show in the CLI
 
 cd /QuestieDB
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)